lib/assessment.rb

```diff
@@ -35,6 +39,7 @@ def every_even(numbers)
   # entries from the words enumerable capitalized.
   def transform(words)
     # Your code goes here
+    words.split.map(&:capitalize)
```

Comment:
Words is presumably already split into an enumerable list, so split is unnecessary. Otherwise correct.